### PR TITLE
Warn if changes to package.json and not yarn.lock

### PR DESCRIPTION
**Pull request by macklinu**
#trivial

Based on https://github.com/facebook/jest/pull/2508

This would have applied to https://github.com/danger/danger-js/pull/101 (discovered when I merged master into and ran `yarn install` on #99).

### CODE REVIEW

**Comment by macklinu** [comment]
I force pushed up an amended commit that would fix the build, but GitHub doesn't seem to be aware of that amended commit yet. Will check back later.

File: dangerfile.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // import { danger, warn } from "danger"
 import fs from "fs"
+import includes from "lodash.includes"
```
Comment (macklinu) [inline dangerfile.js]: Do we need to worry about Node 4 support, or should I just use the built-in `Array.prototype.includes()` function here?

**Comment by macklinu** [comment]
It looks like there are Appveyor builds queueing up too. I logged in with GitHub but don't see a button to cancel old builds.

image

**Comment by orta** [comment]
Yeah, I've been seeing it on other repos - I'm cool with merging without it

and yeah, I don't think TS backports the includes function, so best use the loads version IMO

**Comment by orta** [comment]
👍

**Pull request merged**